File: Java/codechef.java
import java.util.*;
public class codechef {
    public static void main(String[]args){
   // int a,b,c,d,t;
    Scanner scn=new Scanner(System.in);
    int t=scn.nextInt();
    while(t--){
    int a=scn.nextInt();
    int b=scn.nextInt();   
int c=scn.nextInt();
int d=scn.nextInt();
if(a<=c&&b<=d){
    System.out.println("POSSIBLE");
}else{
    System.out.println("IMPOSSIBLE");
}
    }
}
}

File: C/codechef.c
#include<stdio.h>
int display(int a,int b,int c,int d){
    if(a<=c&&b<=d){
        printf("POSSIBLE\n");
    }else{
        printf("IMPOSSIBLE\n");
    }
}
int main(){
    int t;
    scanf( "%d",&t );
    while(t--){
        int a,b,c,d;
        scanf("%d%d%d%d",&a,&b,&c,&d);
        display(a,b,c,d);
    }
    return 0;
}